kotlinx-collections-immutable/src/main/kotlin/kotlinx/collections/immutable/extensions.kt

```diff
@@ -29,140 +29,333 @@ import kotlinx.collections.immutable.implementations.persistentOrderedSet.Persis
 //@Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
 //inline fun <T> @kotlin.internal.Exact ImmutableCollection<T>.mutate(mutator: (MutableCollection<T>) -> Unit): ImmutableCollection<T> = builder().apply(mutator).build()
 // it or this?
+/**
+ * Builds new instance of the persistent set with the provided modifications applied.
+ * Mutable set passed to the [mutator] closure has the same content as this persistent set.
+ */
 inline fun <T> PersistentSet<T>.mutate(mutator: (MutableSet<T>) -> Unit): PersistentSet<T> = builder().apply(mutator).build()
+/**
```

Comment:
Need an empty line between the previous function and this comment

Reply:
+1